src/backends/kimchi/mod.rs

```diff
@@ -292,7 +297,7 @@ impl Backend for KimchiVesta {
 
         let zero = VestaField::zero();
 
-        let _ = &self.add_generic_gate(
+        let () = &self.add_generic_gate(
```

Comment:
just remove the `let () = `

Reply:
fixed in [8b1beaa](https://github.com/zksecurity/noname/pull/100/commits/8b1beaaed30d6aeeb66fd3405869720f4de8d6de)